src/components/Books/BooksList.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { removeBook } from '../../redux/books/books';
+import { removeExistBook } from '../../redux/books/books';
 
 const BooksList = () => {
-  const books = useSelector((state) => state.booksReducer);
+  const books = useSelector((state) => state.booksReducer.books);
```

Comment:
- [ ] The books do not load when you refresh the page, please make sure a server requests is made in the `componentDidMount lifecycle` method or with `react hooks`.